editor/src/core/model/margin-and-padding.ts

```diff
@@ -33,10 +33,25 @@ export const PaddingPropertyPaths: Array<PropertyPath> = [
   PP.create('style', 'paddingTop'),
 ]
 
+export const BackgroundPropertyPaths: Array<PropertyPath> = [
+  PP.create('style', 'background'),
+  PP.create('style', 'backgroundAttachment'),
+  PP.create('style', 'backgroundClip'),
+  PP.create('style', 'backgroundColor'),
+  PP.create('style', 'backgroundOrigin'),
+  PP.create('style', 'backgroundPosition'),
+  PP.create('style', 'backgroundRepeat'),
+  PP.create('style', 'backgroundSize'),
```

Comment:
`backgroundImage`, `backgroundBlendMode`, `backgroundPositionX`, and `backgroundPositionY` if we want to be exhaustive

Reply:
Thanks! Added them in https://github.com/concrete-utopia/utopia/pull/4270/commits/31e48c4382fb6f0dbf39fba6e0b47dde8b59147c